neuror/main.py

```diff
@@ -335,6 +335,7 @@ def run(self,
         for section in sorted(cut_sections_in_bounding_cylinder, key=section_path_length):
             type_ = self.repair_type_map[section]
             if not self.repair_flags.get(type_, True):
+                L.debug('Skipping repair because of the flag for type: %s', type_)
```

Comment:
`Repair flag set to False for section type %s :  --> Skipping repair.`

Reply:
It definitely is. Thanks

```suggestion
                L.debug('Repair flag set to False for section type %s : --> Skipping repair.', type_)
```